### DISCUSSION TITLE
Getting ERR_UNKNOWN_FILE_EXTENSION when running the steps on the documentation

### DISCUSSION THREAD

**Discussion by obedmhg** [percy/cli#881] [Support] [answered]
I am following the steps at https://docs.percy.io/docs/percy-snapshot 

### Steps to replicate:

Node Version: *14.x 16.x*

run 
```
npm install --save-dev @percy/cli
```

using this `snapshots.yml`
```
- name: Home page
  url: https://percy.io/
```

having `.percy.yml`
```
version: 2
snapshot:
  authorization:
    username: admin
    password: admin
```

When I run `npx percy snapshot snapshots.yml`

I get this error
And I am getting this exception:
```
[percy] TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for D:\shared\apps\percy-rb\node_modules\@percy\agent\dist\commands\exec.d.ts
```

Please let me know if I am missing something or this is actually a bug

**Comment by Robdel12** [comment, marked as answer]
Hey @obedmhg! Looks like you have `@percy/agent` (deprecated & replaced by `@percy/cli`) hanging around somewhere: `\node_modules\@percy\agent\dist\`. If you don't have `@percy/agent` installed somewhere, I'd double check to make sure `npx` isn't pulling down and using the `percy` package from NPM.

**Reply by obedmhg** [reply]
I started from scratch and removed my `node_modules` and that did the work! thanks for the help @Robdel12
